include/z64scene.h

```diff
@@ -9,187 +9,187 @@
+    /* 0x4 */ void* segment;
+} SCmdColHeader; // size = 0x8
 
 typedef struct {
-    /* 0x00 */ u8  code;
-    /* 0x01 */ u8  num;
-    /* 0x04 */ void* segment;
-} SCmdRoomList;
+    /* 0x0 */ u8  code;
+    /* 0x1 */ u8  num;
+    /* 0x4 */ void* segment;
+} SCmdRoomList; // size = 0x8
 
 typedef struct {
-    /* 0x00 */ u8  code;
-    /* 0x01 */ u8  data1;
-    /* 0x02 */ UNK_TYPE1 pad2[2];
-    /* 0x04 */ s8  west;
-    /* 0x05 */ s8  vertical;
-    /* 0x06 */ s8  south;
-    /* 0x07 */ u8  clothIntensity;
-} SCmdWindSettings;
+    /* 0x0 */ u8  code;
+    /* 0x1 */ u8  data1;
+    /* 0x2 */ UNK_TYPE1 pad2[2];
+    /* 0x4 */ s8  west;
+    /* 0x5 */ s8  vertical;
+    /* 0x6 */ s8  south;
+    /* 0x7 */ u8  clothIntensity;
+} SCmdWindSettings; // size = 0x8
 
 typedef struct {
-    /* 0x00 */ u8  code;
-    /* 0x01 */ u8  data1;
-    /* 0x04 */ void* segment;
-} SCmdEntranceList;
+    /* 0x0 */ u8  code;
+    /* 0x1 */ u8  data1;
+    /* 0x4 */ void* segment;
+} SCmdEntranceList; // size = 0x8
 
 typedef struct {
-    /* 0x00 */ u8  code;
-    /* 0x01 */ u8  cUpElfMsgNum;
-    /* 0x04 */ u32 subKeepIndex;
-} SCmdSpecialFiles;
+    /* 0x0 */ u8  code;
+    /* 0x1 */ u8  cUpElfMsgNum;
+    /* 0x4 */ u32 subKeepIndex;
+} SCmdSpecialFiles; // size = 0x8
 
 typedef struct {
-    /* 0x00 */ u8  code;
-    /* 0x01 */ u8  gpFlag1;
-    /* 0x04 */ u32 gpFlag2;
-} SCmdRoomBehavior;
+    /* 0x0 */ u8  code;
+    /* 0x1 */ u8  gpFlag1;
+    /* 0x4 */ u32 gpFlag2;
+} SCmdRoomBehavior; // size = 0x8
 
 typedef struct {
-    /* 0x00 */ u8  code;
-    /* 0x01 */ u8  data1;
-    /* 0x04 */ void* segment;
-} SCmdMesh;
+    /* 0x0 */ u8  code;
+    /* 0x1 */ u8  data1;
+    /* 0x4 */ void* segment;
+} SCmdMesh; // size = 0x8
 
 typedef struct {
-    /* 0x00 */ u8  code;
-    /* 0x01 */ u8  num;
-    /* 0x04 */ void* segment;
-} SCmdObjectList;
+    /* 0x0 */ u8  code;
+    /* 0x1 */ u8  num;
+    /* 0x4 */ void* segment;
+} SCmdObjectList; // size = 0x8
 
 typedef struct {
-    /* 0x00 */ u8  code;
-    /* 0x01 */ u8  num;
-    /* 0x04 */ void* segment;
-} SCmdLightList;
+    /* 0x0 */ u8  code;
+    /* 0x1 */ u8  num;
+    /* 0x4 */ void* segment;
+} SCmdLightList; // size = 0x8
 
 typedef struct {
-    /* 0x00 */ u8  code;
-    /* 0x01 */ u8  data1;
-    /* 0x04 */ void* segment;
-} SCmdPathList;
+    /* 0x0 */ u8  code;
+    /* 0x1 */ u8  data1;
+    /* 0x4 */ void* segment;
+} SCmdPathList; // size = 0x8
 
 typedef struct {
-    /* 0x00 */ u8  code;
-    /* 0x01 */ u8  num;
-    /* 0x04 */ void* segment;
-} SCmdTransiActorList;
+    /* 0x0 */ u8  code;
+    /* 0x1 */ u8  num;
+    /* 0x4 */ void* segment;
+} SCmdTransiActorList; // size = 0x8
 
 typedef struct {
-    /* 0x00 */ u8  code;
-    /* 0x01 */ u8  num;
-    /* 0x04 */ void* segment;
-} SCmdLightSettingList;
+    /* 0x0 */ u8  code;
+    /* 0x1 */ u8  num;
+    /* 0x4 */ void* segment;
+} SCmdLightSettingList; // size = 0x8
 // Cloudmodding has this as Environment Settings
```

Comment:
```suggestion
} SCmdLightSettingList; // size = 0x8
```
I don't think that comment is needed